cmd/network/bridge/daemonset.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    name: device-plugin-network-bridge
+  name: device-plugin-network-bridge
+spec:
+  selector:
+    matchLabels:
+      name: device-plugin-network-bridge
+  template:
+    metadata:
+      labels:
+        name: device-plugin-network-bridge
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - name: device-plugin-network-bridge
+        image: phoracek/device-plugin-network-bridge:latest
```

Comment:
probably kubevirt/…

Reply:
i had problems pushing built images to my test kubernetes environment so i chose to be lazy and used docker repo instead. will fix it